svgnative/src/SVGParserArcToCurve.cpp

```diff
@@ -106,7 +106,7 @@ void ArcToCurve(SVGNative::Path& path, float startX, float startY, float radiusX
         return;
     }
 
-    SVG_ASSERT(radiusX == radiusY); // untested
+    //SVG_ASSERT(radiusX == radiusY); // untested
```

Comment:
Why remove this assertion? Float comparison should include epsilon, but in general?